In [3]:
import pandas as pd

In [4]:
df = pd.read_csv("Resume.csv")

In [5]:
print(df.shape)      # rows and columns

(2484, 4)


In [6]:
print(df.head()) 

         ID                                         Resume_str  \
0  16852973           HR ADMINISTRATOR/MARKETING ASSOCIATE\...   
1  22323967           HR SPECIALIST, US HR OPERATIONS      ...   
2  33176873           HR DIRECTOR       Summary      Over 2...   
3  27018550           HR SPECIALIST       Summary    Dedica...   
4  17812897           HR MANAGER         Skill Highlights  ...   

                                         Resume_html Category  
0  <div class="fontsize fontface vmargins hmargin...       HR  
1  <div class="fontsize fontface vmargins hmargin...       HR  
2  <div class="fontsize fontface vmargins hmargin...       HR  
3  <div class="fontsize fontface vmargins hmargin...       HR  
4  <div class="fontsize fontface vmargins hmargin...       HR  


In [7]:
print(df.columns)    # column names

Index(['ID', 'Resume_str', 'Resume_html', 'Category'], dtype='str')


In [8]:
df

,ID,Resume_str,Resume_html,Category
0,16852973,HR ADMINISTRATOR/MARKETING ASSOCIATE\...,"<div class=""fontsize fontface vmargins hmargin...",HR
1,22323967,"HR SPECIALIST, US HR OPERATIONS ...","<div class=""fontsize fontface vmargins hmargin...",HR
2,33176873,HR DIRECTOR Summary Over 2...,"<div class=""fontsize fontface vmargins hmargin...",HR
3,27018550,HR SPECIALIST Summary Dedica...,"<div class=""fontsize fontface vmargins hmargin...",HR
4,17812897,HR MANAGER Skill Highlights ...,"<div class=""fontsize fontface vmargins hmargin...",HR
...,...,...,...,...
2479,99416532,RANK: SGT/E-5 NON- COMMISSIONED OFFIC...,"<div class=""fontsize fontface vmargins hmargin...",AVIATION
2480,24589765,"GOVERNMENT RELATIONS, COMMUNICATIONS ...","<div class=""fontsize fontface vmargins hmargin...",AVIATION
2481,31605080,GEEK SQUAD AGENT Professional...,"<div class=""fontsize fontface vmargins hmargin...",AVIATION
2482,21190805,PROGRAM DIRECTOR / OFFICE MANAGER ...,"<div class=""fontsize fontface vmargins hmargin...",AVIATION


## Data Dictionary

| Column | What it means | Example |
|--------|--------------|---------|
| ID | Unique ID number for each resume | 16852973 |
| Resume_str | Full plain text of the resume | "HR ADMINISTRATOR/MARKETING ASSOCIATE..." |
| Resume_html | Same resume in HTML format (with styling tags) | `<div class="fontsize...` |
| Category | Job category the resume belongs to | HR, AVIATION, Data Science |

## Dataset Summary
- Total resumes: 2484
- Total columns: 4
- Job categories include: HR, AVIATION and more

In [9]:
print("Total Resumes:", df.shape[0])
print("Total Columns:", df.shape[1])
print("\nJob Categories:")
print(df['Category'].value_counts())

Total Resumes: 2484
Total Columns: 4

Job Categories:
Category
INFORMATION-TECHNOLOGY    120
BUSINESS-DEVELOPMENT      120
ADVOCATE                  118
CHEF                      118
FINANCE                   118
ENGINEERING               118
ACCOUNTANT                118
FITNESS                   117
AVIATION                  117
SALES                     116
HEALTHCARE                115
CONSULTANT                115
BANKING                   115
CONSTRUCTION              112
PUBLIC-RELATIONS          111
HR                        110
DESIGNER                  107
ARTS                      103
TEACHER                   102
APPAREL                    97
DIGITAL-MEDIA              96
AGRICULTURE                63
AUTOMOBILE                 36
BPO                        22
Name: count, dtype: int64


In [11]:
df.isna().sum().sum()

np.int64(0)

In [12]:
df.duplicated().sum()

np.int64(0)